content/php/concepts/date-functions/terms/date-parse/date-parse.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: 'date_parse()'
+Description: 'Accepts a date as a parameter, parses it and, retruns information about the given date in the form of an array.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Time'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The `date_parse()` is an inbuilt function in PHP which is used to find the detailed information about a specified date.
+
+## Syntax
+
+```pseudo
+date_parse($date);
+```
+
+The `$date` parameter is mandatory. The `date_parse()` function returns an associative array of detailed information for a specified date on success and returns FALSE on failure.
+
+## Codebyte Example
```

Comment:
I'd like to see an additional regular example with an output block that uses the various named parameters, maybe printing them out individually.

At least, I'd like to see this codebyte changed to a regular example with an output block afterwards, like:
````
This results in the following output:

```shell
Array
(
    [year] => 2023
    [month] => 6
    [day] => 27
    [hour] => 11
    [minute] => 27
    [second] => 40
    [fraction] => 0.7
    [warning_count] => 0
    [warnings] => Array
        (
        )
    [error_count] => 0
    [errors] => Array
        (
        )
    [is_localtime] => 
)
```
````

Reply:
@SSwiniarski 

noted on this

thanks

Reply:
@SSwiniarski 

i did a pull and a push

thanks